app/scripts/modules/core/src/filterModel/FilterCollapse.tsx

```diff
@@ -13,24 +13,31 @@ export class FilterCollapse extends React.Component<{}> {
 
   public render() {
     return (
-      <>
+      <div className="filters-toggle layer-high sp-margin-s-xaxis">
         <h3 className="filters-placeholder">
           <Tooltip value="Show filters">
-            <a className="btn btn-xs btn-default pin clickable" onClick={() => this.onClick(true)}>
+            <a
```

Comment:
while you're in here, is it possible to change this to a `<button>` tag? an `a` tag styled as a button should be used only if it's a link to somewhere else while actions should use a `<button>` tag.  This comment applies to elsewhere in this file.

Reply:
Yes, will do that.